dns/pfSense-pkg-bind/files/usr/local/pkg/bind.xml

```diff
@@ -290,6 +290,12 @@
 			<type>checkbox</type>
 			<enablefields>bind_forwarder_ips</enablefields>
 		</field>
+		<field>
+			<fielddescr>DNSSEC Validation</fielddescr>
+			<fieldname>bind_dnssec_validation</fieldname>
+			<description>Enable recursive BIND server to perform DNSSEC validation.</description>
```

Comment:
Seems a little weirdly worded.

Maybe this: "Enable DNSSEC validation when acting as a recursive BIND server"
Or this: "Enable DNSSEC validation when BIND is acting as a recursive resolver"

Reply:
Ok, fixed